app/sidekiq/central_mail/submit_central_form686c_job.rb

```diff
@@ -28,19 +30,19 @@ def perform(saved_claim_id, encrypted_vet_info, encrypted_user_struct)
       Rails.logger.info('CentralMail::SubmitCentralForm686cJob running!',
                         { user_uuid: user_struct['uuid'], saved_claim_id:, icn: user_struct['icn'] })
       @claim = SavedClaim::DependencyClaim.find(saved_claim_id)
-      @claim.add_veteran_info(vet_info)
+      claim.add_veteran_info(vet_info)
```

Comment:
what necessitated changing this from an instance var to a local one? (seeking to understand)

Reply:
Nothing necessitated it. I just have been leaning towards this style lately. I believe one thing that is nice about using `attr_reader` is that it makes it easier to track where the var has been/can be assigned